x-pack/plugins/security_solution/public/detections/components/rules/severity_badge/index.tsx

```diff
@@ -10,19 +10,21 @@ import React from 'react';
 
 import { euiLightVars } from '@kbn/ui-shared-deps-src/theme';
 import { HealthTruncateText } from '../../../../common/components/health_truncate_text';
+
+const { euiColorVis0, euiColorVis5, euiColorVis7, euiColorVis9 } = euiLightVars;
+const severityToColorMap: Record<string, string> = {
+  low: euiColorVis0,
+  medium: euiColorVis5,
+  height: euiColorVis7,
+};
+
 interface Props {
   value: string;
 }
 
 const SeverityBadgeComponent: React.FC<Props> = ({ value }) => {
   const displayValue = upperFirst(value);
-  const color = 'low'
-    ? euiLightVars.euiColorVis0
-    : value === 'medium'
-    ? euiLightVars.euiColorVis5
-    : value === 'high'
-    ? euiLightVars.euiColorVis7
-    : euiLightVars.euiColorVis9;
+  const color = severityToColorMap[value] ?? euiColorVis9;
```

Comment:
And here it could be `const color = severityToColorMap[value] ?? <some default color like gray that's used in the "Last response" column for unknown statuses>;`

Reply:
using `subdued` color, which is default 'greyed out' color